kernel/src/misc/wm/wm.c

```diff
@@ -435,14 +435,25 @@ wm_window_t* wm_window_at(int32_t x, int32_t y) {
 }
 
 void wm_draw_mouse(rect_t old, rect_t new) {
-    wm_refresh_partial(old);
+
+
+    if((old.top != new.top) || (old.left != new.left))
+    	wm_refresh_partial(old);
```

Comment:
Remove the whole `if` and refresh call (leave no blank lines!), and add a call to `wm_refresh_partial(prev)` above [this line](https://github.com/the-grue/SnowflakeOS/blob/master/kernel/src/misc/wm/wm.c#L551). This avoids the extra `if`, which is already done there in that `wm_mouse_callback` function. Also could you remove the `old` argument to `wm_draw_mouse` in the process?

Reply:
Done and done.